CesiumUtility/include/CesiumUtility/ExtensibleObject.h

```diff
@@ -118,5 +118,37 @@ struct CESIUMUTILITY_API ExtensibleObject {
    * experimental, or next-version properties.
    */
   JsonValue::Object unknownProperties;
+
+  /**
+   * @brief Calculates the size in bytes of this ExtensibleObject, including all
+   * of its extras and extensions. Calling this method may be slow as it
+   * requires traversing the entire object.
+   */
+  int64_t getSizeBytes() const {
+    int64_t accum = 0;
+    accum += sizeof(ExtensibleObject);
+    accum += this->extensions.bucket_count() *
+             (sizeof(std::string) + sizeof(JsonValue));
+    for (auto& [k, v] : this->extensions) {
+      accum += k.capacity() * sizeof(char) - sizeof(std::string);
+      const JsonValue* pValue = std::any_cast<JsonValue>(&v);
+      accum += pValue->getSizeBytes() - sizeof(JsonValue);
```

Comment:
Well, this is a pain. A generic dispatcher like the one I envisioned when I wrote the above is difficult. We'd need to generate a dispatch function that does an `any_cast` to each of the possible extensions in turn, and then calls the provided callback for the first cast that succeeds. Not terribly efficient.

Or we do what the "writer" system does and look up classes based on the name of the extension and the name of the class extended (because a single logical glTF extension can have a different C++ type for extending different glTF classes).

Either way, it's a little tricky to figure out where to put this. It can't go in `ExtensibleObject` because that's in CesiumUtility, so we can't generate glTF-specific code there. So we'd have to pass in some kind of (generated) extension context to ExtensibleObject's `getSizeBytes` I guess. `ExtensibleObject::getSizeBytes` would have to be a template on the type of the extension context.

I think what we should do for now is:
1. Completely remove the extension size tracking from `ExtensibleObject::getSizeBytes`.
2. Update the generate doc for `getSizeBytes` to say it doesn't include the size of any extensions.

And I think that will be good enough. If we find that extension size tracking is actually important in the future, we can invest the time to do something like the above.